interface-elements-for-web/articles/rich-text-editor/viewing-and-navigating/document-views.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Document Views
+
+---
+# Document Views
+Use ribbon buttons in **View** tab's **Document Views** group to switch document views.
+
+![EUD_ASPxRichEdit_ViewTab](../../../images/img117731.png)
+
+The **Simple View** displays a document in the entire editing area and ignores the page layout. In this mode, the editor does not split a document into pages and ignores headers, footers, and page layout settings: margins, orientation, size, columns, and breaks.
+
+The **Print Layout** displays a document such as it will be printed: split into pages, with headers, footers, and with applied page layout settings: margins, orientation, size, columns, and breaks.
```

Comment:
"such as it will"?

Reply:
changed to 'The **Print Layout** displays a document split into pages, with headers,...'